qcoptions.py

```diff
@@ -8,7 +8,7 @@
                   help="Minimum value of scan parameter", metavar="scanmin", default=0)
 parser.add_option("--scanmax", type="int", dest="scanmax",
                   help="Maximum value of scan parameter", metavar="scanmax", default=254)
-parser.add_option("--vfatmask", type="int", dest="vfatmask",
+parser.add_option("--vfatmask", type="int", dest="vfatmask", 
```

Comment:
I added some statements to my ~/.vimrc to remove trailing white space on *.py files whenever the save action is performed by vim (e.g. `:w`).  Hopefully this should solve the problem...but git may not have realized anything in the file has changed.